tracing-core/src/subscriber.rs

```diff
@@ -167,6 +167,33 @@ pub trait Subscriber: 'static {
     /// [`register_callsite`]: #method.register_callsite
     fn enabled(&self, metadata: &Metadata<'_>) -> bool;
 
+    /// Returns the highest [verbosity level][level] that this `Subscriber` will
+    /// enable, or `None`, if the subscriber does not implement level-based
+    /// filtering or chooses not to implement this method.
+    ///
+    /// If this method returns a [`Level`][level], it will be used as a hint to
+    /// determine the most verbose level that will be enabled. This will allow
+    /// spans and events which are more verbose than that level to be skipped
+    /// more efficiently. Subscribers which perform filtering are strongly
```

Comment:
I never liked "verbosity" when applied to levels, but... I guess that ship sailed. I was always confused by what it meant.

Reply:
If there's another wording you think is clearer, I'm all ears! If we're going to change it, we should change it throughout the docs.